src/routes/productRouter.ts

```diff
@@ -69,4 +70,7 @@ router.get("/seller-get-products",userAuthorization(["seller"]),isSellerShopExis
 
 router.get("/user-get-products",isPaginated, productController.userGetProducts);
 
+router.get("/buyer-get-product/:id",userAuthorization(["buyer"]),isProductAvailable,productController.viewSpecificProduct)
```

Comment:
this is your task's description.
As a `User`
I should be able to retrieve a given item.
As a `Seller `I should be able to see an item if it is from my
collection

As a` Buyer`, I should be able to see any item from the available items.
then why only `buyer` is authorized  unless we have another endpoint for `seller` and others

Reply:
previously they told me to just create one for the buyer